lib/rpc/endpoints/get_health.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Soroban.RPC.GetHealth do
+  @moduledoc """
+  GetHealth request implementation for Soroban RPC.
+  """
+  @behaviour Soroban.RPC.Endpoint.Spec
+
+  alias Soroban.RPC.{GetHealthResponse, Request}
+
+  @endpoint "getHealth"
+
+  @impl true
+  def request(params \\ nil)
+
+  def request(nil) do
```

Comment:
Just keep one function clause... 
```suggestion
  def request(params \\ nil) do
```